src/gui/DatabaseWidget.h

```diff
@@ -91,6 +91,7 @@ class DatabaseWidget : public QStackedWidget
     Group* currentGroup() const;
     bool canDeleteCurrentGroup() const;
     bool isGroupSelected() const;
+    bool currentGroupHasChildren() const;
```

Comment:
This function is not specific to DatabaseWidget and should be removed. If anything the Group class should have a "hasChildren()" function.